code/game/world.dm

```diff
@@ -104,6 +103,7 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 		spawn(0)
 			to_chat(world, "<span class='boldannounce'>Rebooting world immediately due to host request</span>")
 		rustg_log_close_all() // Past this point, no logging procs can be used, at risk of data loss.
+		TgsReboot()
```

Comment:
This was done in this PR to simplify things down the line.

TLDR; TGS uses the `TgsReboot()` signal if youre on the basic watchdog to kill the DD process. In our current config, it would kill right at end round before anything had been saved if we were in basic mode. This fixes that.